auth/src/main/java/com/firebase/ui/auth/ui/phone/SubmitConfirmationCodeFragment.java

```diff
@@ -134,7 +134,7 @@ public void onSaveInstanceState(Bundle outState) {
 
     private void setTimer(long millisUntilFinished) {
         mCountDownTextView.setText(String.format(getString(R.string.resend_code_in),
-                timeRoundedToSeconds(millisUntilFinished)));
+                                                 timeRoundedToSeconds(millisUntilFinished)));
```

Comment:
If this was auto-formatted, can we change the rules not to do these aligned hanging indents (here and everywhere)?  They don't survive refactoring.